src/main/groovy/de/fntsoftware/gradle/MarkdownToPdf.groovy

```diff
@@ -1,28 +1,28 @@
 package de.fntsoftware.gradle
 
 class MarkdownToPdf {
-	private static final DEFAULT_RESOURCE = 'bitbucket.css';
+	private static final DEFAULT_RESOURCE = 'bitbucket.css'
 
-	String cssFile;
+	String cssFile
 
-	private String cssFileContent = null;
+	private String cssFileContent = null
 
 	public String getCssFileContent() {
 		if(this.cssFileContent == null) {
```

Comment:
Can you please also add a whitespace here.

Reply:
I missed that one, yes. I'll fix it.